indicators/slurm.py

```diff
@@ -179,10 +181,31 @@ def parse_args():
     ) = parse_args()
 
     # make batch files for each model / scenario / variable combination
-    sbatch_dir = out_dir.joinpath("/slurm/indicators")
+    sbatch_dir = out_dir.joinpath("slurm")
     _ = [fp.unlink() for fp in sbatch_dir.glob("*.slurm")]
     sbatch_dir.mkdir(exist_ok=True)
 
+    # make QC dir and "to-do" list for each model / scenario / indicator combination
+    # the "w" accessor should overwrite any previous qc.txt files encountered
+    qc_dir = out_dir.joinpath("qc")
+    qc_dir.mkdir(exist_ok=True)
+    qc_file = qc_dir.joinpath("qc.csv")
+    with open(qc_file, "w") as q:
```

Comment:
Is this statement just to overwrite / create a blank `qc_file`?

Reply:
Yep. For every run of `slurm.py`, we get a new QC file. I think is OK to overwrite each time, since they are just lists of filepaths. But I did consider putting a timestamp on the `qc_error.txt` file, so we could keep those without overwriting.